README.md

```diff
@@ -1,2 +1,359 @@
+    <li><a href="https://www.javascript.com/">Javascript</a></li>
+    <li><a href="https://create-react-app.dev/">React</a></li>
+    <li><a href="https://reactrouter.com/">React Router</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server Side / Back-End</summary>
+  <ul>
+    <li><a href="https://www.json.org/json-en.html">JSON</a></li>
+    <li><a href="https://nodejs.org/en/">Node.js</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://www.iqair.com/">Air Quality API</a></li>
+  </ul>
+</details>
+
+<details>
+    <summary>Package, Library, Framework</summary>
+      <ul>
+        <li><a href="https://jestjs.io/">Jest</a></li>
+        <li><a href="https://axios-http.com/docs/intro">axios</a></li>
+        <li><a href="https://redux-toolkit.js.org/">Reduxjs Toolkit</a></li>
+      </ul>
+  </details>
+
+<!-- Features -->
+### Tools i have used for this project <a name="tools"></a>
+  <details>
+    <summary>Code Convention, Code Analysis</summary>
+      <ul>
+        <li><a href="https://eslint.org/">ESLint</a></li>
+        <li><a href="https://webhint.io/">Webhint</a></li>
+        <li><a href="https://stylelint.io/">Stylelint</a></li>
+        <li><a href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?hl=en">Lighthouse</a></li>
+        <li><a href="https://www.npmjs.com/package/npm-check">node_modules checker</a></li>
+      </ul>
+  </details>
+  <details>
+    <summary>Version Control, CI/CD, Hosting Service</summary>
+      <ul>
+        <li><a href="https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow">GitFlow</a></li>
+        <li><a href="https://pages.github.com/">Github Pages</a></li>
+        <li><a href="https://github.com/features/actions">Github Actions</a></li>
+        <li><a href="https://git-scm.com/">Git</a></li>
+        <li><a href="https://vercel.com/">Vercel</a></li>
+      </ul>
+  </details>
+  <details>
+    <summary>IDE, Desktop Apps, Other Tools</summary>
+      <ul>
+        <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+        <li><a href="https://desktop.github.com/">Github Desktop</a></li>
+        <!-- <li><a href="https://www.figma.com/">Figma</a></li> -->
+      </ul>
+  </details>
+
+### Key Features <a name="key-features"></a>
+
+- display a list of games 
+- display a air quality and weather informations based on selected game
+- single page apps (SPA)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name=""></a>
+
+<!-- > Add a link to your deployed project. -->
+- Click the following url for [... Live Demo ...](https://free-pc-games.vercel.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Preview
+[![Watch the video](https://cdn.loom.com/sessions/thumbnails/d51acfe30a5748d3a444a81b6bbd5e86-with-play.gif)](https://www.loom.com/embed/d51acfe30a5748d3a444a81b6bbd5e86)
```

Comment:
- [ ] Great job making your presentation video. However, the project requirement about the presentation video hasn't completely met. You're yet to highlight any pieces of code that you're very proud of. This simply means that you're expected to show some line of code in your text editor that responsible for a feature in your project, not to just say it alone while presenting your project in the video. Show some line of code in your vscode text editor and show  the feature that line of code responsible for on your page. Kindly do that.

[Your Video](https://www.loom.com/embed/d51acfe30a5748d3a444a81b6bbd5e86)

Reply:
only have 5 minutes for the presentation and also show some lines of code, ok.

Reply:
I'll try it